From 1cef1ddc28f327534cfdb433d1f2e10d01b10f44 Mon Sep 17 00:00:00 2001
From: Jakub Filak <jfilak@redhat.com>
Date: Thu, 24 Jul 2014 16:07:36 +0200
Subject: [PATCH] json: include json.h accordingly to json-c CFLAGS

We use $(pkg-config --cflags json-c) but we include <json/json.h>. The
proper way to include the json-c header is to include <json.h>.

That absolute include causes errors while compiling with json-c-0.12
where the base include directory was changed  to "json-c".

https://github.com/json-c/json-c/blob/master/ChangeLog

Signed-off-by: Jakub Filak <jfilak@redhat.com>
---
 src/lib/json.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/lib/json.c b/src/lib/json.c
index 08ffe8c..fc26bc6 100644
--- a/src/lib/json.c
+++ b/src/lib/json.c
@@ -17,7 +17,7 @@
     51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 */
 
-#include <json/json.h>
+#include <json.h>
 
 #include <satyr/abrt.h>
 #include <satyr/report.h>
-- 
1.9.3

